packages/server/modules/core/repositories/streams.ts

```diff
@@ -983,6 +987,19 @@ export async function revokeStreamPermissions(params: {
 }) {
   const { streamId, userId } = params
 
+  const existingPermission = await StreamAcl.knex()
+    .where({
+      [StreamAcl.col.resourceId]: streamId,
+      [StreamAcl.col.userId]: userId
+    })
+    .first()
+  if (!existingPermission) {
+    // User already doesn't have permissions
+    return await Streams.knex<StreamRecord>()
```

Comment:
This allows invoking the repo fn even if the user already doesn't have any kind of role. Useful in tests